### ISSUE TITLE
Android does not have WebView.NavigationType.other

### ISSUE THREAD

**Issue by gunars** [jerolimov/react-native-showdown#3]
**Problem:**
On Android there is an error, because Android does not have WebView.NavigationType.other. Error message: "undefined is not an object (evaluating '_reactNative.WebView.NavigationType.other') Android does not have WebView.NavigationType.other there"

**Solution:**
in the showdown markdownview.js
onNavigationStateChange, should be like:

``` javascript
onNavigationStateChange(navState) {
// Check if user pressed on a link
if(navState.url !== 'about:blank' ) {
LinkingIOS.openURL(navState.url);
this.setState({
navigationStateChange: this.state.navigationStateChange + 1
});
}
}
```

**Comment by philohelp** [comment]
linkingIOS is deprecated and should be replaced by linking.

**Comment by donedgardo** [comment]
I have made a pull request [Pull request 9](https://github.com/jerolimov/react-native-showdown/pull/9)
#9

**Comment by christoph-jerolimov** [comment]
Thanks for the PR. I merged #7 which fixed this issue. If not please reopen the issue. Thanks for contributing.
